src/Calculator.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const Calculator = () => (
+  <div className="table">
+    <div className="row field"><p id="field-text">Welcome</p></div>
+    <div className="row">
+      <div className="button gray">AC</div>
+      <div className="button gray">+/-</div>
+      <div className="button gray">%</div>
+      <div className="button orange">/</div>
+    </div>
+    <div className="row">
+      <div className="button gray">7</div>
+      <div className="button gray">8</div>
+      <div className="button gray">9</div>
+      <div className="button orange">X</div>
+    </div>
+    <div className="row">
+      <div className="button gray">4</div>
+      <div className="button gray">5</div>
+      <div className="button gray">6</div>
+      <div className="button orange">-</div>
+    </div>
+    <div className="row">
+      <div className="button gray">1</div>
+      <div className="button gray">2</div>
+      <div className="button gray">3</div>
+      <div className="button orange">+</div>
+    </div>
+    <div className="row">
+      <div className="big-button gray">0</div>
+      <div className="button gray">.</div>
+      <div className="button orange">=</div>
+    </div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] There should be a Calculator.js file inside the components directory.
- [ ] The Calculator.js component should be class-based
![storage5](https://user-images.githubusercontent.com/55635977/133340584-305a85e5-337a-441b-a9eb-4ddabcef0871.png)

- [ ] The App.js component should be class-based.
- [ ] The App.js(main component) should have the imported the Calculator.js component 

!![storage1](https://user-images.githubusercontent.com/55635977/133340895-86180f82-aa1b-4c3c-8d0f-01723d582d5e.png)